### PR TITLE
[Bugfix][Android] Prevent empty notifications when receiving Firebase data messages

### PR DESCRIPTION
**Pull request by Robfz**
## Current behavior
When receiving Firebase data messages (messages without `title` or `body`), an empty notification is shown.

## New behavior
When receiving  Firebase data messages, the notification is only handled by JS and no notification is shown.

### Other changes
- Tests for the changes above.
- Fixed a duplicated test that wasn't testing what it should (`onReceived_validDataForBackgroundApp_postNotificationAndNotifyJs`)

### CODE REVIEW

**Comment by Robfz** [comment]
@yogevbd is there any feedback on this, or is there anything I could do to drive this PR forward? Thanks.

**Comment by douira** [comment]
why is this marked as stale if the maintainers have just not looked at it yet

**Comment by CesarLanderos** [comment]
@yogevbd can we move this forward please? 🙏 , we are kind of blocked, we also want to push other fixes but we have seen no activity in this PR for so many months that we feel a little bit discouraged, this is the best library that we have found and we very much like to contribute to make it even better 🙏

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-notifications",
+  "name": "@hiretend/react-native-notifications",
```
Comment (yogevbd) [inline package.json:2]:
```suggestion
  "name": "react-native-notifications",
```

File: package.json [hunk 2]
```diff
@@ -74,11 +74,11 @@
     "@types/lodash": "4.14.153"
   },
   "publishConfig": {
-    "registry": "https://registry.npmjs.org/"
+    "registry": "https://npm.pkg.github.com/"
```
Comment (yogevbd) [inline package.json:77]:
```suggestion
    "registry": "https://registry.npmjs.org/"
```

File: package.json [hunk 3]
```diff
@@ -74,11 +74,11 @@
...
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/wix/react-native-notifications.git"
+    "url": "ssh://git@github.com/hiretend/react-native-notifications.git"
```
Comment (yogevbd) [inline package.json:81]:
```suggestion
    "url": "https://github.com/wix/react-native-notifications.git"
```

**Review by yogevbd — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (yogevbd) [inline package.json:2]:
```suggestion
  "name": "react-native-notifications",
```

File: package.json (hunk 2 above)
Comment (yogevbd) [inline package.json:77]:
```suggestion
    "registry": "https://registry.npmjs.org/"
```

File: package.json (hunk 3 above)
Comment (yogevbd) [inline package.json:81]:
```suggestion
    "url": "https://github.com/wix/react-native-notifications.git"
```

**Review by yogevbd — CHANGES_REQUESTED** [review]

**Comment by Robfz** [comment]
@yogevbd sorry about the extra commits! We made further changes to our fork. I created a fresh PR with only the changes related to this bug (https://github.com/wix/react-native-notifications/pull/677). Thanks for reviewing!

**Pull request closed without merging**